src/axom/quest/tests/quest_intersection_shaper.cpp

```diff
@@ -755,14 +755,15 @@ dimensions: 3
 
 //---------------------------------------------------------------------------
 // Define testing functions for different modes.
-#if defined(RUN_AXOM_SEQ_TESTS)
+#if defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE)
```

Comment:
Minor: Do the SEQ and OpenMP tests really require Umpire? 
I would expect the GPU testes to require Umpire, but not the CPU-based tests.

Reply:
Yes.  The SEQ and OpenMP tests go into `replacementRuleTestSet()` and eventually through `prepareShapeQueryImpl()`, whose definition at `IntersectionShaper.hpp` line 362 is within `#if defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE)`.

Reply:
If we wanted to run intersection shaping sequentially, we would need to pull that out and roll the reducers and for_all loops without RAJA.